From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Giuseppe Maggio <jertlok@proton.me>
Date: Sun, 9 Aug 2026 17:24:24 +0200
Subject: [PATCH] dt-bindings: sound: qcom,sm8250: add the msm8998 sound card

The msm8998 machine driver binds on "qcom,msm8998-sndcard" and shares
the properties this schema already describes for the other q6dsp-based
cards.

Assisted-by: LLM
Signed-off-by: Giuseppe Maggio <jertlok@proton.me>
---
 Documentation/devicetree/bindings/sound/qcom,sm8250.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml b/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
index dae440ecab59..d0e9d4271609 100644
--- a/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
+++ b/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
@@ -34,6 +34,7 @@ properties:
           - fairphone,fp5-sndcard
           - qcom,apq8096-sndcard
           - qcom,glymur-sndcard
+          - qcom,msm8998-sndcard
           - qcom,qcm6490-idp-sndcard
           - qcom,qcs615-sndcard
           - qcom,qcs6490-rb3gen2-sndcard
